.gitignore

```diff
@@ -6,3 +6,5 @@ test/*.js
 test/integration/*.js
 !test/ts-node-register.js
 .DS_Store
+ts_src/*.js
+ts_src/payments/*.js
```

Comment:
what? remove! in `ts_src` there are no js files. If so, you're doing something really weird.

Reply:
I'm using a vscode extension to debug that generates file in ts_src. That's just for confort :thinking:

Reply:
This is something that concerns your workspace... I don't have any js file in my ts_src folder, but in general, no js files should be present in there at all.

Please, revert these changes.